man/journalctl.xml

```diff
@@ -92,8 +92,8 @@
     disabled; see the <option>--no-pager</option> option and the "Environment" section below.</para>
 
     <para>When outputting to a tty, lines are colored according to priority: lines of level ERROR and higher
-    are colored red; lines of level NOTICE and higher are highlighted; lines of level DEBUG are colored
-    lighter grey; other lines are displayed normally.</para>
+    are colored red; lines of level WARNING are colored yellow; lines of level NOTICE and higher are highlighted;
```

Comment:
"NOTICE and higher" is a bit misleading. The next higher level is WARNING, and that already uses a different color. Hence simply replace "NOTICE and higher" → "NOTICE".

Reply:
Done. But should we then also change the "other lines" term to "lines of level INFO" as other is only INFO?

Reply:
true, please do.

I guess we use a different color for all levels, except for ERR/CRIT/ALERT/EMERG which all get the same red color.